modules/environment-roles/templates/github_assume_role.json.tpl

```diff
@@ -12,11 +12,7 @@
           "token.actions.githubusercontent.com:aud": "sts.amazonaws.com"
         },
         "ForAnyValue:StringLike": {
-          "token.actions.githubusercontent.com:sub": [
-            "repo:nationalarchives/tdr-*",
-            "repo:nationalarchives/tna-custodian*",
```

Comment:
Can probably add `repo:nationalarchives/da-reference-generator*` and  `repo:nationalarchives/tna-custodian*` as a local variable in this repo and concat that into the `repo_names` instead of defining it in the tdr-configurations repo.